Video/VideoIndexer/README.md

```diff
@@ -0,0 +1,176 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-search
+azureDeploy: https://raw.githubusercontent.com/Azure-Samples/azure-search-power-skills/master/Video/VideoIndexer/azuredeploy.json
+name: "Sample skill for enabling video indexing"
+description: "This custom skill will invoke the Azure Video Indexer, placing a simplified insights model back into Blob Storage. You can then trigger another indexer to merge the insights back into your main search index."
+---
+
+# Video Indexer
+
+This custom skill will invoke the [Azure Video Indexer](https://docs.microsoft.com/en-us/azure/media-services/video-indexer/), placing a simplified insights model back into Blob Storage. You can then trigger another indexer to merge the insights back into your main search index.
+
+## Requirements
+
+This skill requires a Video Indexer account, and key to function.
+
+## Settings
+
+This function requires the following application settings.
+
+| Setting Name | Description |
+| ---- | --- |
+| MediaIndexer_AccountId | Azure Video Indexer Account Id |
+| MediaIndexer_Location | Azure Video Indexer Account location, e.g. trial |
+| MediaIndexer_AccountKey | Azure Video Indexer Account key |
+| MediaIndexer_StorageConnectionString | Azure Storage Connection string pointing to a blob store to store simplified Video Indexer results |
+| MediaIndexer_StorageContainer | Name of the container to place the simplified Video Indexer results |
+| MediaIndexer_CallbackFunctionCode | Azure Function code enabling the Video Indexer to invoke the callback function |
+
+
+## Deployment
+
+[![Deploy to Azure](https://azuredeploy.net/deploybutton.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fgraemefoster%2Fazure-search-power-skills%2Ffeature%2Fvideo-indexing-skill%2FVideo%2FVideoIndexer%2Fazuredeploy.json)
+
+## hello-world
+
+### Sample Input:
+
+```json
+{
+    "values": [
+        {
+            "recordId": "r1",
+            "data":
+            {
+              "metadata_storage_path": "<SAFE-BASE64-ENCODED-PATH-TO-VIDEO>",
+              "metadata_storage_name": "<VIDEO-NAME>"
+            }
+        }
+    ]
+}
+```
+
+### Sample Output:
+
+```json
+{
+    "values": [
+        {
+            "recordId": "r1",
+            "data": {
+            },
+            "errors": [],
+            "warnings": []
+        }
+    ]
+}
+```
+
+### Sample Insights Blob Output
```

Comment:
Could you add a sentence explaining what this is? I'm assuming this is what ends up in blob storage which is then indexed?

Reply:
Added better documentation